addons/pos_restaurant/models/pos_config.py

```diff
@@ -133,3 +134,169 @@ def post_install_pos_localisation(self, companies=False):
             })
             pos_configs.setup_defaults(company)
             super(PosConfig, self).post_install_pos_localisation(companies)
+
+    def action_restaurant_create_closed_and_open_pos_session_demo(self):
```

Comment:
This function is for demo data only. Isn't there another way to do this so as not to overload our models?